### DISCUSSION TITLE
User setting for a cleaner tray menu

### DISCUSSION THREAD

**Discussion by mdcclxv** [File-New-Project/EarTrumpet#1048] [Ideas]
Hi there,

This is definitely no biggie, but a nice to have. I would like to be able to remove the highlighted part of the items in the tray menu. See the attached screenshot.

![image](https://user-images.githubusercontent.com/33668915/151986132-48b1ca77-942c-4771-87c4-1679f971cec3.png)

Thanks,
Katy

**Comment by davuxcom** [comment]
This is the device name in Windows. You can dig into the settings in Windows audio and rename the devices if you like. A couple notes about that though:
- There's no way to reset the old name, so take note of it
- They reset frequently on Windows upgrades and such.

**Comment by mdcclxv** [comment]
Yes, I know it is the device name. As the title reads, I would like a cleaner look of the popup menu, that's all. I don't want to rename the device, I just want it gone from the popup, if that's not much trouble. A checkbox in the app settings reading "show device name in tray menu".

Thanks,
Katy.
